ros/ros1/Utils.hpp

```diff
@@ -57,12 +57,15 @@ inline auto GetTimestampField(const PointCloud2::ConstPtr msg) {
 
 // Normalize timestamps from 0.0 to 1.0
 inline auto NormalizeTimestamps(const std::vector<double> &timestamps) {
-    const double max_timestamp = *std::max_element(timestamps.cbegin(), timestamps.cend());
-    // check if already normalized
-    if (max_timestamp < 1.0) return timestamps;
+    const auto min_max_it = std::minmax_element(timestamps.cbegin(), timestamps.cend());
```

Comment:
use structured bidnings     `const auto [a,b] = std::minmax_element(timestamps.cbegin(), timestamps.cend());`

Reply:
Fix in new commit